splitgraph/resources/static/audit_trigger.sql

```diff
@@ -139,7 +139,6 @@ CREATE OR REPLACE FUNCTION splitgraph_audit.audit_table (
     RETURNS void
     AS $body$
 DECLARE
-    stm_targets text = 'INSERT OR UPDATE OR DELETE OR TRUNCATE';
```

Comment:
Do you still need to change this file if we're not enabling audit triggers on the LQ checkout?

Reply:
No, just initially saw that the variable was obsolete so I removed it. Do you prefer that I revert this regardless?

Reply:
Oh, if it's actually unused, then it's fine to delete this.